warehouse/models/staging/audit/stg_audit__cloudaudit_googleapis_com_data_access.sql

```diff
@@ -0,0 +1,95 @@
+{{
+    config(
+        materialized='incremental',
+        incremental_strategy='insert_overwrite',
+        partition_by={
+            'field': 'date',
+            'data_type': 'date',
+            'granularity': 'day',
+        },
+        partitions=['current_date()'],
+        cluster_by='job_type',
+    )
+}}
+-- we should use https://cloud.google.com/bigquery/docs/reference/standard-sql/json_functions#parse_json when available
+
+-- Note these two source CTEs a direct reference instead of source, because a new table is created daily
+WITH latest AS (
+    {% set today = modules.datetime.date.today() %}
+
+    SELECT *
+    FROM cal-itp-data-infra.audit.cloudaudit_googleapis_com_data_access_{{ today.strftime('%Y%m%d') }}
+),
+
+everything AS (
```

Comment:
any chance we should add the is_incremental check higher up here to bypass any of these CTEs / make it clearer up front what path we're going to follow? I know that they don't actually get evaluated unless needed, and I like the `latest` / `everything` naming but just wondering

Reply:
Eh technically we could but bigquery will ignore CTEs that aren't ultimately used, and I think it's useful to use the CTEs as the "switch" but I could be convinced otherwise.